gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/distributed/message/GfxdFunctionMessage.java

```diff
@@ -1820,4 +1824,9 @@ public final long getRootMessageSendTime() {
   // execution to be on all copies of data to gather stats etc.
   public void setSendToAllReplicates(boolean includeAdmin) {
   }
+
+  public void setAbortOnLowMemory(boolean flag) {
+    abortOnLowMemory = flag;
+  }
+
 }
```

Comment:
Have set only for StatementExecutorMessage/ PrepStatementExecutorMessage. Don't think it fails for region.delete convertibles. Will verify.